src/layouts/MetricsLayout.tsx

```diff
@@ -54,6 +58,17 @@ export const MetricsLayout = ({
         },
       ],
     );
+
+    if (["TALLYING" || "RESULTS"].includes(roundState!)) {
+      navLinks.push(
+        ...[
+          {
+            href: `/${domain}/admin/distribute`,
+            children: `Distribute`,
+          },
+        ],
+      );
+    }
```

Comment:
Why is Distribute shown on the MetricsLayout? It will render for everyone?

Reply:
@hussedev asked to be included and it shows up only when you are admin.